packages/create-catalyst/src/utils/user-agent.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  isDevelopment,
+  isLinux,
+  isMacOS,
+  isProduction,
+  isTest,
+  isWindows,
+  nodeVersion,
+  process,
+  provider,
+  runtime,
+} from 'std-env';
+
+import packageInfo from '../../package.json';
+
+const { name, version } = packageInfo;
+
+const getOS = () => {
+  if (isWindows) return 'Windows';
+  if (isMacOS) return 'macOS';
+  if (isLinux) return 'Linux';
+
+  return 'Unknown OS';
+};
+
+const getEnv = () => {
+  if (isDevelopment) return 'Development';
+  if (isTest) return 'Test';
+  if (isProduction) return 'Production';
+
+  return 'Unknown Env';
+};
```

Comment:
When would the CLI be running in Test/Production? Or are we just adding them because we can? (with `std-env`)

Reply:
CI/CD